src/main/java/uk/gov/hmcts/reform/et/syaapi/helper/TseApplicationHelper.java

```diff
@@ -34,6 +36,10 @@ public static String formatCurrentDate(LocalDate date) {
         return date.format(UK_LOCAL_DATE_PATTERN);
     }
 
+    public static String getCurrentDateTime() {
```

Comment:
Java docs if keeping this based on comments further down

Reply:
Java Docs updated